lib/views/git-tab-view.js

```diff
@@ -32,7 +34,30 @@ export default class GitTabView {
   }
 
   render() {
-    if (this.props.repository.showGitTabInit()) {
+    if (this.props.repository.isTooLarge()) {
+      const workingDir = this.props.repository.getWorkingDirectoryPath();
+      return (
+        <div className="github-Panel is-empty" tabIndex="-1">
+          <div ref="noRepoMessage" className="github-Panel no-repository">
+            <div className="large-icon">
+              <GitLogo />
+            </div>
+            {workingDir === os.homedir()
+              ? <div className="initialize-repo-description">
+                  The number of changed files in the repository at <strong>{workingDir}</strong> is
+                  too large for Atom to manage. In addition, the repository is located at your
+                  home directory, which may be unintended. <a href="https://gist.github.com/BinaryMuse/something">More details</a>
+              </div>
+              : <div className="initialize-repo-description">
+                  The number of changed files in the repository at <strong>{workingDir}</strong> is
+                  too large for Atom to manage. Ensure that the repository path is correct and
```

Comment:
Can you help me understand what it means to "ensure that the repository path is correct?" Is there a scenario where the repository path would be incorrect?

Reply:
Yeah I agree that this wording is a bit unclear. I _think_ it was mostly referring to the case when the user accidentally had a .git folder in their home directory. Which is addressed in the other case so we can probably just drop this wording here. @BinaryMuse 💭s?